packages/node-cli/messages.json

```diff
@@ -246,13 +245,15 @@
 	"ERRORS_GENERAL_CONNECTION_PROBLEM": "There was an error when communicating with the server. Try again later.",
 	"ERRORS_MISSING_DEFAULT_AUTH_ID": "The {0} property is missing.",
 	"ERRORS_NO_OBJECTS": "There are no objects matching your search criteria. Select a different set of object types, and if applicable, check the specified application ID and script ID.",
+	"ERRORS_NOT_PROJECT_FOLDER": "No {0} file was found in {1}. Run \"{2}\" in a valid project folder.{3}For more information about the project structure, see https://system.netsuite.com/app/help/helpcenter.nl?fid=section_155931221634.html.",
```

Comment:
Please add the link to the ApplicationConstants. There is LINKS section, you can add another section like INFO -> PROJECT_STRUCTURE. And you'll be able to reuse it also in "ERRORS_SEE_PROJECT_STRUCTURE"